lib/utils.js

```diff
@@ -144,3 +153,57 @@ export async function storeError(errorMsg) {
 
   await update(queryError);
 }
+
+
+export async function updateStatusJob(jobUri, status) {
```

Comment:
isn't there already a function for this in the code, just asking

Reply:
I don't think so, there is a cleanup job i think but it deletes the job instead of updating the status i think